scripts/plot_dataframe.py

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python3
 
 import matplotlib  # Install PYQT5 manually if you want to test this helper function
-matplotlib.use("Qt5Agg")
 import matplotlib.pyplot as plt
-
 from freqtrade import exchange, analyze
 
+matplotlib.use("Qt5Agg")
```

Comment:
I'm afraid that `matplotlib` needs to be initialized before importing `matplotlib.pyplot`, otherwise it won't be used.